src/main.py

```diff
@@ -20,13 +20,12 @@ async def app_exception_handler(_, exc):
         },
     )
 
-
 @app.on_event("startup")
 async def start_schedule():
     import threading
 
-    threading.Thread(target=register_student_first_half).start()
-    threading.Thread(target=register_student_second_half).start()
+    threading.Thread(target=register_student_first_half, daemon=True).start()
+    threading.Thread(target=register_student_second_half, daemon=True).start()
```

Comment:
@maecco Pode explicar um pouquinho o porque dessa alteração?

Reply:
@vianaz Claro, em poucas palavras essa kword faz com que se o programa principal for derrubado as threads sao derrubadas tambem.
https://www.geeksforgeeks.org/python-daemon-threads/

Reply:
@maecco poderia colocar um comentário explicando e contendo esse link no código? por favor

Reply:
@vianaz Claro, feito!